### PR TITLE
Move websocket-client to a script module

### PR DESCRIPTION
**Pull request by enen92**
Closes https://github.com/Montellese/mediaimporter.plex/issues/5

I tested it under your last available OSX build (with python3) and it works well. Unfortunately the last py2 compatible build was already removed from the mirrors and I was unable to test it with python2.

I transfered the ownership of https://github.com/enen92/script.module.websocket_client to you

### CODE REVIEW

File: addon.xml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="mediaimporter.plex" name="Plex Media Importer" version="0.0.1" provider-name="Montellese">
+<addon id="mediaimporter.plex" name="Plex Media Importer" version="0.0.2" provider-name="Montellese">
     <requires>
-        <import addon="xbmc.python" version="2.1.0"/>
+        <import addon="xbmc.python" version="2.26.0"/>
```
Comment (enen92) [inline addon.xml:4]: This was introduced since you test builds are based on Matrix (api version is 3.0.0). 2.26.0 is the API version for Leia.

**Review by Montellese — APPROVED** [review]
Looks great thanks.

**Comment by Montellese** [comment]
I've started test builds of the old state with Python 2.7 at https://jenkins.kodi.tv/view/Automation/job/BuildMulti-Choice/330/.

**Comment by enen92** [comment]
Cool, I'm trying my best at https://github.com/Montellese/mediaimporter.plex/issues/2 now :)

**Comment by enen92** [comment]
Changed the alert.py as per your suggestion (https://github.com/Montellese/mediaimporter.plex/commit/46383bfb5b22207e5f570f9953f18d4807624675). 
Gave it a run in your last build (py2.7) and the one I had installed (py3) and it works in both versions as expected 💯

**Comment by Montellese** [comment]
I have added `script.module.websocket_client` to `repository.mediaimport` see https://github.com/Montellese/repository.mediaimport/commit/f23e40cf0d3e70d02a4b3883b46dc0133692bc4f.

**Comment by Montellese** [comment]
Thanks a lot for your work and support!

**Pull request merged**